src/test/addrman_tests.cpp

```diff
@@ -387,22 +387,22 @@ BOOST_AUTO_TEST_CASE(addrman_getaddr)
     CNetAddr source1 = ResolveIP("250.1.2.1");
     CNetAddr source2 = ResolveIP("250.2.3.3");
 
-    // Test 23: Ensure GetAddr works with new addresses.
+    // Test: Ensure GetAddr works with new addresses.
     addrman.Add(addr1, source1);
     addrman.Add(addr2, source2);
     addrman.Add(addr3, source1);
     addrman.Add(addr4, source2);
     addrman.Add(addr5, source1);
 
     // GetAddr returns 23% of addresses, 23% of 5 is 1 rounded down.
-    BOOST_CHECK(addrman.GetAddr().size() == 1); 
+    BOOST_CHECK_EQUAL(addrman.GetAddr().size(), 1); 
```

Comment:
Please no trailing whitespace. This will cause issues in version control when used with editors that remove trailing whitespace by default.